Vertikal description list

pethel opened the pull request:
Vi har behov for en vertikal description list. 

![image](https://user-images.githubusercontent.com/2248579/57620226-732f6a00-7588-11e9-9547-0d4d96d13074.png)

hdaljord commented on packages/ffe-lists-react/src/DescriptionList.js:
```diff
@@ -23,6 +24,8 @@ DescriptionList.propTypes = {
     /** Large list. Can't be used with `medium` prop */
     large: bool,
+    /** Displays dts and dds inline*/
+    horizontal: bool,
```
Please add this to the index.d.ts file as well (with a ?, since it's optional)

pethel replied:
https://github.com/SpareBank1/designsystem/pull/632/commits/edca781aeb0019122ca5c1bdb53f41a97c0d1c3e

henrikhermansen reviewed (dismissed):
At first it seemed strange to make a breaking change to accommodate these needs.

However, on second thought, to not make a breaking change here would require us to add styling which would essentially be reverting the styling we already have. This breaking change makes a lot more sense.

I approve the changes, but it would be great if you could also update the markdown files with some relevant examples, so it will show on design.sparebank1.no.

Commit:
feat(ffe-lists-react): added vertical description list

BREAKING CHANGE: The description list is now vertical by default. To get old behaviour 'horizontal' prop must be passed.

Commit:
feat(ffe-lists): added vertical description list

BREAKING CHANGE: The description list is now vertical by default. To get old behaviour 'horizontal' class modifier must be used

ivarni commented:
[peerDependencies](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-lists-react/package.json#L44) should also be updated since the change was made breaking.
